### PR TITLE
Fix some issues with NFT code samples

### PR DESCRIPTION
**Pull request by mDuo13**
This is just a quick band-aid the code sample problems noted in #2600.

- Fix the file extension of JS code sample
- Handle error case when buy offers aren't found
- Add a `package.json` file
- Add a brief README for the code samples

### CODE REVIEW

File: _code-samples/non-fungible-token/js/list-nft-pages-and-buy-offers.js [hunk 1]
```diff
@@ -47,6 +50,6 @@
     }
     client.disconnect()
     // End main()
-  }
-  
-  main()
+}
+
+main()
```
Comment (justinr1234) [inline _code-samples/non-fungible-token/js/list-nft-pages-and-buy-offers.js:54]: If this is running in nodejs I think the program would end with an unresolved promise

Comment (intelliot) [reply _code-samples/non-fungible-token/js/list-nft-pages-and-buy-offers.js:54]: Thanks, I think I fixed this in c829b7d

**Comment by intelliot** [comment]
I tried to "suggest changes" but github doesn't allow doing that on lines which weren't touched by this PR, so I just pushed my changes as a new commit. Feel free to revert if it looks bad, but I think my edits are all improvements :)

**Review by intelliot — APPROVED** [review]
I did `npm i` and `node list-nft-pages-and-buy-offers.js` and it appears to work fine. Seems like there are currently no buy offers for these NFTs though.

**Pull request merged**